modules/react/common/lib/utils/colorUtils.ts

```diff
@@ -1,6 +1,13 @@
-import {colors} from '@workday/canvas-kit-react/tokens';
+import {CanvasColor, colors} from '@workday/canvas-kit-react/tokens';
 import chroma from 'chroma-js';
 
+export function getColor(value?: CanvasColor | string) {
```

Comment:
Can we add some jsdocs here? I think this is a valuable function, but would love some documentation here as well!

Reply:
@mannycarrera4 Let me know if there's anything else you'd like me to add here.